python/cuml/linear_model/linear_regression.pyx

```diff
@@ -200,6 +200,8 @@ class LinearRegression(LinearPredictMixin,
     fit_intercept : boolean (default = True)
         If True, LinearRegression tries to correct for the global mean of y.
         If False, the model expects that you have centered the data.
+    copy_X : bool, default=True
+        If True, X will be copied; else, it may be overwritten.
```

Comment:
I would recommend adding a bit more context that it'll be modified in the `svd` option, as well as a quick note of additional GPU memory consumption.

Reply:
I've expanded the documentation a bit, but did not explicitly mention the SVD algorithm since that seems to be an implementation detail to me. Please have a look and let me know if you would still prefer the explicit mention, thanks!